src/pages/SudoPrompt.vue

```diff
@@ -9,15 +9,24 @@
     <p>
       Rancher Desktop requires administrative credentials ("sudo access") in
       order to provide a better experience.  We would like to have access for
-      the following:
+      the following reasons:
     </p>
-    <ul>
-      <li v-for="item in explanations" :key="item" class="monospace" v-text="item" />
+    <ul class="reasons">
+      <li v-for="(paths, reason) in explanations" :key="reason">
+        <details>
+          <summary>{{ SUDO_REASON_DESCRIPTION[reason].title }}</summary>
+          <p>{{ SUDO_REASON_DESCRIPTION[reason].description.replace(/\s+/g, ' ') }}</p>
+          <p>This will modify the following paths:</p>
+          <ul>
+            <li v-for="path in paths" :key="path" class="monospace" v-text="path" />
+          </ul>
+        </details>
+      </li>
     </ul>
     <p>
       We will display the actual prompt once this window is closed.  Cancelling
-      the password prompt will cause Rancher Desktop to run in reduced
-      functionality mode, but it should still start.
+      the password prompt will cause Rancher Desktop to run with reduced
+      functionality, but it should still start.
```

Comment:
I know it's not part of this PR, but I think we can drop `but it should still start`. Letting people know that Rancher Desktop will run with reduce functionality should be plenty.

Reply:
Dropped the extra phrase in 93be988fc29fd4dab6a545b03f86460a75d38993.